src/gui/statsdialog.cpp

```diff
@@ -98,7 +98,7 @@ void StatsDialog::updateUI() {
                                                   : "0"));
   // Disk queues
   ui->labelQueuedJobs->setText(QString::number(cs.jobQueueLength()));
-  ui->labelJobsTime->setText(QString::number(cs.averageJobTime()));
+  ui->labelJobsTime->setText(tr("%1 ms", "%1 milliseconds").arg(cs.averageJobTime()));
```

Comment:
`tr("%1 ms", "18 milliseconds")`. Remember the second argument is an example string for the translators usually.